impeller/display_list/dl_dispatcher.cc

```diff
@@ -1104,7 +1104,8 @@ void DlDispatcher::drawDisplayList(
 void DlDispatcher::drawTextBlob(const sk_sp<SkTextBlob> blob,
                                 SkScalar x,
                                 SkScalar y) {
-  const auto text_frame = TextFrameFromTextBlob(blob);
+  const auto text_frame = TextFrameFromTextBlob(
```

Comment:
I wonder if this will work against @dnfield s work to defer computing the text scale. Seems like this is strictly an improvement for now, but we should probably also defer the initial scale value?

Reply:
Yeah, good catch. Using the scale here could indeed cause problems. Text frames drawn through DrawPicture might not end up having precise enough glyph bounds depending on how much the elements of the picture get scaled up.

We can get away with just using a decently large number here instead of the scale, so I went ahead and removed the usage and checked to make sure it's still working with unreasonably huge glyphs on Android.

Reply:
After doing this, there are no longer any overlapping characters in the `MaybeHasOverlapping` test. The accompanying comment was:
> Characters probably have overlap due to low fidelity text metrics, but this could be fixed.

So if there happens to be an accompanying bug I guess we can close that too. :)

Reply:
Ahh interesting. So we don't actually end up rasterizing at a higher scale, we're essentially just tricking Skia into giving higher fidelity metrics.

I don't think I filed a bug for that particular overlap issue, I would update the expectation and delete the TODO.